src/app/shared-data.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from '@angular/core';
```

Comment:
What is this service for? Can it be removed since we have a dataService already that's not mocked?